Generate/src/Medication-9-0-7/_components/assert-returnEntryCount.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="../../../general/schematron/NictizTestScript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <nts:component  xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript">
+    <nts:parameter name="breakdown" value=""/>
```

Comment:
Het zou denk ik wel goed zijn om in een commentaartje parameters te documenteren. In ieder geval voor "common" dingen.

Reply:
Ga ik doen